tests/test_modeling_common.py

```diff
@@ -60,6 +60,7 @@
     MODEL_FOR_SEQUENCE_CLASSIFICATION_MAPPING_NAMES,
     MODEL_FOR_TOKEN_CLASSIFICATION_MAPPING_NAMES,
     MODEL_FOR_VIDEO_CLASSIFICATION_MAPPING_NAMES,
+    MODEL_FOR_VISION_2_SEQ_MAPPING_NAMES,
```

Comment:
One question here about mapping. I see that "VISION_2_SEQ_MAPPING" does not contain all of vision-language models that can generate. Is it supposed to be that way or we should update it? Right now it is missing Idefics and Fuyu